data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,11 @@
 [
   {
+    "id": "forging_standard",
+    "type": "requirement",
+    "//": "Forging of steel items (per steel chunk)",
+    "tools": [ [ [ "forge", 20 ], [ "char_forge", 20 ], [ "oxy_torch", 20 ] ] ]
```

Comment:
> See #17303.

Which part? And what does it imply?
Oxy torches are currently quite rare and expensive to use. Keeping that 4x multiplier would be good.
Alternatively, they could get 4x as many charges in each tank.